tests/osproc/tworkingdir.nim

```diff
@@ -0,0 +1,13 @@
+discard """
+  file: "tworkingdir.nim"
+  output: ""
+"""
+
+import osproc, os
+
+let dir1 = getCurrentDir()
+var process = startProcess("/usr/bin/true", "/usr/bin")
```

Comment:
While this code is from my issue, I wouldn't use it in a test case because it is platform-dependent. `/usr/bin/env true` would be better here.